lib/src/json_schema/json_schema.dart

```diff
@@ -735,6 +746,9 @@ class JsonSchema {
   /// Base URI of the ID. All sub-schemas are resolved against this
   Uri _idBase;
 
+  /// Something like and ID, but different.
```

Comment:
We could probably be somewhat more helpful with this description :)

Reply:
oh man! this was supposed to be a temporary description....